tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -7,7 +7,7 @@
 from .test_import_rke2_cluster import (
     RANCHER_RKE2_VERSION, create_rke2_multiple_control_cluster
 )
-from .test_rke_cluster_provisioning import rke_config
+from .test_rke_cluster_provisioning import AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, rke_config
```

Comment:
We have environment variables of type Azure Resource Manager (ARM), these should have the same value as the `AZURE_CLIENT_ID` `AZURE_CLIENT_SECRET`

```python
    client_id = os.environ.get('ARM_CLIENT_ID', '')
    client_secret = os.environ.get('ARM_CLIENT_SECRET', '')
```

Reply:
Would we then have to have to export `ARM_CLIENT_ID` and `ARM_CLIENT_SECRET` instead? I currently source `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET`, should that not be the case?